src/runtime/main.ts

```diff
@@ -39,10 +40,19 @@ export function getProcessedCss(
   const tpaParams = generateTPAParams(siteColors, siteTextPresets, styleParams, options);
 
   const processor = getProcessor({cssVars: injectedData.cssVars, plugins});
+  const {isPermissive} = options;
 
   return injectedData.customSyntaxStrs.reduce((processedContent, part) => {
-    const newValue = processor.process({part, tpaParams});
-
+    let newValue;
+    try {
+      newValue = processor.process({part, tpaParams});
+    } catch (e) {
+      if (isPermissive) {
```

Comment:
Let's change that prop to `strictMode` it should be with default value `true`, and you will pass `false`.